easybuild/easyconfigs/m/MEGAHIT/MEGAHIT-1.1.2-foss-2018a.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'MakeCp'
+
+name = 'MEGAHIT'
+version = '1.1.2'
+
+homepage = 'https://github.com/voutcn/megahit'
+description = """An ultra-fast single-node solution for large and complex
+ metagenomics assembly via succinct de Bruijn graph"""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = ['https://github.com/voutcn/%(namelower)s/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['d0d3965dd49c6fdaea958ef66146cb6b30b7d51acbbfe94194c437f15a424cb5']
+
+# This version is CPU-only.  It's also possible to compile this with
+# CUDA, but that isn't working yet (and requires libcuda.so!).
+
+dependencies = [
+    ('Python', '2.7.14'),
+]
+
+# for GPU, 'use_gpu=1 abi=1' at least, plus CUDA dep above
+buildopts = 'version=v%(version)s verbose=1 use_gpu=0'
+
+# for GPU, more binaries?
+files_to_copy = [
+    (['%(namelower)s',
+      '%(namelower)s_asm_core',
+      '%(namelower)s_sdbg_build',
+      '%(namelower)s_toolkit'],
+     'bin'),
```

Comment:
minor style suggestion:

```python
files_to_copy = [
    (['%(namelower)s', '%(namelower)s_asm_core', '%(namelower)s_sdbg_build', '%(namelower)s_toolkit'], 'bin'),
    'LICENSE',
    'README.md',
]

Reply:
I considered this, but had the idea that multi-element lists were to be one-per-line.  (In fact, you requested that in https://github.com/easybuilders/easybuild-easyconfigs/pull/5698.)  Is there an easy rule-of-thumb to remember?

Reply:
The format I suggested makes it clearer that there's basically 3 things being copied: binaries, LICENSE & README.

There's no clear rule for this, it's mostly intuition (and potential a bike-shedding opportunity).